plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -109,6 +110,55 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    @Override
+    public void testAddColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAddColumn);
+    }
+
+    @Override
+    public void testDropColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testDropColumn);
+    }
+
+    @Override
+    public void testAlterTableAddLongColumnName()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAlterTableAddLongColumnName);
+    }
+
+    @Override
+    protected void testAddAndDropColumnName(String columnName, boolean delimited)
+    {
+        skipSchemaChangeOperationLockInterruptedError(() -> super.testAddAndDropColumnName(columnName, delimited));
+    }
+
+    @Override
+    protected String dataMappingTableName(String trinoTypeName)
```

Comment:
Why do you override this method?

Reply:
Ignite can not parse the table name that has two consecutive wildcard. I filed a [pr](https://github.com/apache/ignite/pull/10579) to fix  but still ongoing.

Reply:
Let's fix the base method instead.